keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -851,6 +857,24 @@ resource. This is the same approach that was chosen for [generic ephemeral
 volumes](https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1698-generic-ephemeral-volumes/README.md).
 For a resource driver there is no difference.
 
+Different lifecycles can be combined with different allocation modes
+arbitrarily. Some combinations are more useful than others:
+
+```
++-----------+----------------------------------------------------------------------+
+|           |                             allocation mode                          |
+| lifecycle |             immediate              |   delayed                       |
++-----------+------------------------------------+---------------------------------+
+| regular   | starts the potentially             | avoids wasting resources        |
+| claim     | slow allocation as soon            | while they are not needed yet   |
+|           | as possible                        |                                 |
++-----------+------------------------------------+---------------------------------+
+| inline    | same benefit as above,             | resource allocated when needed, |
```

Comment:
Can we / should we just make Inline + Immediate be a validation error?

Reply:
We could. But the code won't benefit from it, so I am not sure whether we should.

It feels like we are making a decision for the user that is theirs to make. Note that "delayed" is the default, so the user would have to choose "immediate" explicitly.